server/src/test/java/io/spine/server/commandbus/AbstractCommandBusTestSuite.java

```diff
@@ -282,4 +293,69 @@ boolean wasHandlerInvoked() {
             return handlerInvoked;
         }
     }
+
+    /**
+     * A configurable {@link SystemGateway} for tests.
+     */
+    protected interface TestSystemGateway extends SystemGateway {
+
+        /**
+         * Schedules the given {@code command}.
+         *
+         * <p>The given command will be accessed by {@link CommandIndex#scheduledCommands()}
+         * from this {@link SystemGateway}.
+         */
+        void schedule(Command command);
+    }
+
+    protected class TestMultitenantSystemGateway implements TestSystemGateway {
```

Comment:
I assume someone is going to be using these `protected` classes someday. Please have the class-level documentation for those.